spec/hash_utils_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe FormatParser::HashUtils do
+  describe '.deep_transform_keys' do
```

Comment:
What happens if the hash being transformed has both `"bb"` and `:bb` keys?

Reply:
From what I saw, it will return the lastest value in the hash.

It is the same behavior as active support, and I don't have a better idea for it, do you have?

I added [a spec](https://github.com/WeTransfer/format_parser/blob/a000efa08c2499b5cd329442f2b69566b1fdcfdf/spec/hash_utils_spec.rb#L35-L39) to cover it!

wdyt?

Reply:
Fine to just have a spec so that we demonstrate what we expect to happen there 👍